pkgs/development/python-modules/screed/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pythonOlder
+, bz2file
+, setuptools
+, setuptools_scm
```

Comment:
I think this is supposed to be `setuptools-scm` rather than `setuptools_scm`, here and in `buildInputs`.

```
nixpkgs-review pr 205878
...
error: Function called without required argument "setuptools_scm" at /home/djm/.cache/nixpkgs-review/pr-205878-2/nixpkgs/pkgs/development/python-modules/screed/default.nix:7, did you mean "setuptools-scm"?
nix --experimental-features nix-command --system aarch64-linux eval --json --impure --expr (import /nix/store/j2jxl8bsxnl5m6y09r96x729xv2zjf0w-nixpkgs-review-2.7.0/lib/python3.10/site-packages/nixpkgs_review/nix/evalAttrs.nix { allowAliases = false; attr-json = /tmp/tmpwtsc9u89; }) failed to run, /tmp/tmpwtsc9u89 was stored inspection
https://github.com/NixOS/nixpkgs/pull/205878 failed to build
```

There is an alias in `pkgs/top-level/python-aliases.nix`, which is presumably why `nix-build` works, but everything else is using `setuptools-scm`

Reply:
Thank you! I followed how the package is named in PyPI and used in python imports, didn't notice the right name in Nixpkgs is with `-`